### PR TITLE
Add recommendations engine js files

### PR DESCRIPTION
**Pull request by nitriques**
- rating.js: required bits to be able to "vote" on a page.
Currently does it on page enter, delayed.
- recommendations.js: required bits to enabled a recommendations listing
to be created with data attributes

### CODE REVIEW

**Comment by nitriques** [comment]
ping @fhamon @PascalPiche @dominic-blain @beaubbe

**Comment by nitriques** [comment]
ping @fhamon @PascalPiche @dominic-blain @beaubbe

**Comment by nitriques** [comment]
Come on guys I've been waiting for 2 months now

File: src/modules/recommendations.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+			imgSrc: !!hit.imgSrc ? hit.imgSrc : '',
+			imgJitFormat: !!hit.imgJitFormat ? hit.imgJitFormat : ''
+		};
+	};
+	
+	var comAlgolia = App.components.create('algolia-search', {
+		resultsCtnSelector: RES,
+		resultsContentSelector: CON,
+		resultsItemTemplateSelector: TEM,
+		algoliaAttributesToRetrieve: [
```
Comment (fhamon) [inline src/modules/recommendations.js]: Could this be in it's own variable so it's easier to edit?

**Pull request merged**